### PR TITLE
edit cmd/kubelet/app/init_windows.go to structured Log

### PR DESCRIPTION
**Pull request by LinshanYu**
/kind documentation

edit cmd/kubelet/app/init_windows.go to structured Log，https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md

### CODE REVIEW

File: cmd/kubelet/app/init_windows.go [hunk 1]
```diff
@@ -52,7 +52,7 @@ func initForOS(windowsService bool, windowsPriorityClass string) error {
 	}
 	kubeletProcessHandle := windows.CurrentProcess()
 	// Set the priority of the kubelet process to given priority
-	klog.Infof("Setting the priority of kubelet process to %s", windowsPriorityClass)
+	klog.InfoS("Setting the priority of kubelet process to %s", windowsPriorityClass)
```
Comment (kikisdeliveryservice) [inline cmd/kubelet/app/init_windows.go:55]: Please see: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#remove-string-formatting-from-log-message specifically: "All string formatting (%d, %v, %w, %s) should be removed and log message string simplified."

This means that the changes in this PR are incomplete and the messages need to be updated as per the guidelines above.

**Review by kikisdeliveryservice — CHANGES_REQUESTED** [review]

**Comment by kikisdeliveryservice** [comment]
This is a user facing change so if it requires a release note, the original post needs to be edited to include text.

**Comment by ehashman** [comment]
/priority backlog
/hold

since this probably needs a release note

**Pull request closed without merging**